packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -18,7 +18,7 @@ export interface ExpandableSectionProps {
   /** Text that appears in the toggle when collapsed (will override toggleText if both are specified; used for uncontrolled expandable with dynamic toggle text) */
   toggleTextCollapsed?: string;
   /** Callback function to toggle the expandable content */
-  onToggle?: () => void;
+  onToggle?: (isOpen: boolean) => void;
```

Comment:
Can we just call this parameter `isExpanded`
Can we make the parameter optional, otherwise this would be a breaking change.